### ISSUE TITLE
Case study project

### ISSUE THREAD

**Issue by sandro-guest** [multar/scm#2]
### TLDR: 

A zero-config monorepo toolchain and sandbox development environment for case studies.

## Initiative 

In order to examine and communicate various subjects (techniques, configurations, etc.), a sandbox environment  with good defaults will help to start an up-close examination of _the subject_.

Simple and explicit functional requirements for a familiar application such as a "to do list manager app" will abstract away domain complexity to get practical knowledge on a specific subject. Then competing solutions can be compared side by side without distractions.

### Hypothesis

Should be helpful in different scenarios, such as

- Examine new framework or library
- Start a discussion
- Compare various techniques
- Learn something 
- Ask for help

Notable "trials" can be marked and archived for later revisions and reference. Should integrate well with common tool set for version control, project management and knowledge organization.

## Acceptance criteria

For launch must have:
 
- A clearly defined project scope (expressed in README)
- Naive implementation of minimal feature set with usage instructions 
- code of conduct communicating intention of initiative
- Contribution guidelines and relevant docs
- Baseline configuration with _standard_ defaults
- A demo case study 

Those can be adjusted later for specific case, with best practices shaping new defaults.

### Useful resources

- [TodoMVC](http://todomvc.com/)
